resources/views/partials/asset-bulk-actions.blade.php

```diff
@@ -19,8 +19,13 @@
         @endcan
         @else
         @can('update', \App\Models\Asset::class)
-            <option value="edit">{{ trans('button.edit') }}</option>
```

Comment:
Small thing, but can you revert the change to the cuddling here? We always want a space between `{{` and `}}` and whatever is inside.